spec/requests/providers/property_value_spec.rb

```diff
@@ -4,83 +4,129 @@
     end
   end
 
   describe 'PATCH /providers/applications/:id/property_value', type: :request do
     let(:params) do
       {
         legal_aid_application: { property_value: 123_456.78 },
-        legal_aid_application_id: application.id
+        legal_aid_application_id: application.id,
+        'continue-button' => 'Continue'
       }
     end
 
-    before { patch providers_legal_aid_application_property_value_path(application), params: params }
-
-    context 'when a property value is entered' do
-      context 'property is mortgaged' do
-        let(:application) { create :legal_aid_application, :with_applicant, :with_own_home_mortgaged }
+    context 'Continue button pressed' do
+      before { patch providers_legal_aid_application_property_value_path(application), params: params }
 
-        # TODO: replace with correct path once other controllers are ready
-        xit 'redirects to the shared question' do
-          # expect(response).to redirect_to shared_question_path
-        end
+      context 'when a property value is entered' do
+        context 'property is mortgaged' do
+          let(:application) { create :legal_aid_application, :with_applicant, :with_own_home_mortgaged }
 
-        # TODO: remove once we can redirect to next controller
-        it 'renders plain message' do
-          expect(response.body).to eq 'Navigate to question 1c: Mortgage'
+          it 'redirects to the outstanding mortgage question' do
+            expect(response).to redirect_to providers_legal_aid_application_outstanding_mortgage_path(application)
+          end
         end
-      end
 
-      context 'property is owned outright' do
-        let(:application) { create :legal_aid_application, :with_applicant, :with_own_home_owned_outright }
+        context 'property is owned outright' do
+          let(:application) { create :legal_aid_application, :with_applicant, :with_own_home_owned_outright }
 
-        # TODO: replace with correct path once other controllers are ready
-        xit 'redirects to the shared question' do
-          # expect(response).to redirect_to mortgage_question_path
+          # TODO: replace with correct path once other controllers are ready
+          it 'redirects to the shared question' do
+            expect(response).to redirect_to providers_legal_aid_application_shared_ownership_path(application)
+          end
         end
 
-        # TODO: remove once we can redirect to next controller
-        it 'renders plain message' do
-          expect(response.body).to eq 'Navigate to question 1d: Shared?'
+        context 'with valid values' do
+          let(:application) { legal_aid_application }
+
+          it 'records the value in the legal aid application table' do
+            legal_aid_application.reload
+            expect(legal_aid_application.property_value).to be_within(0.01).of(123_456.78)
+            expect(legal_aid_application.updated_at.utc.to_i).to be_within(1).of(Time.now.to_i)
```

Comment:
Again - I don't think the update time check is needed.

Reply:
These are all old tests - let's remove them as a separate exercise.